packages/peer-discovery-mdns/test/multicast-dns.spec.ts

```diff
@@ -141,15 +142,6 @@ describe('MulticastDNS', () => {
     await stop(mdnsC)
   })
 
-  it('should start and stop with go-libp2p-mdns compat', async () => {
```

Comment:
Why remove this test?

Reply:
Historically go-libp2p-mdns didn't speak spec-compliant mDNS so we had a compatibility layer.

go-libp2p-mdns has since been updated to behave better so the compatibility layer was removed, this test should have gone too.